client/src/screens/TagsResultPage/TagsResultPage.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import Navbar from '../../components/SearchResultPage/Navbar/Navbar'
+import classes from './TagsResultPage.module.css'
+import gql from 'graphql-tag'
+import { Query } from 'react-apollo'
+import { Link } from 'react-router-dom'
+
+export default function SearchResultPage({ match, props }) {
+    const SearchTags = gql`
+    query{
+    Tag(name: "${match.params.query}"){
+        pathways{
+            name
+            id
+            tags{
+              name
+            }
+          }
+          }
+        }`
+
+    return (
+        <div className={classes.searchResults}>
+            <Navbar />
+            <h3 className={classes.heading}>
+                Showing similar tag results for "{match.params.query}"
+            </h3>
+
+            <Query query={SearchTags}>
+                {({ loading, error, data }) => {
+                    if (loading)
+                        return <h3 className={classes.heading}>Loading</h3>
+                    if (error)
+                        return (
+                            <h3 className={classes.heading}>
+                                Error! {error.message}
+                            </h3>
+                        )
+                    console.log(data)
+                    return (
```

Comment:
There is some issue with the results. When I click on a particular tags, the correct results are shown, but from this screen if I again click on a tag, the screen shows no results, even though there should be. Please look into it